ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml

```diff
@@ -89,6 +89,36 @@ tests:
     test:
     - ref: addon-tests-rhods
     workflow: addons-and-operators
+- as: smoke-ipi-rhods-addon-baseline-fips
+  cron: 0 12 * * 6
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      ADDON1_CONFIG: name=managed-odh;notification-email=interop-qe-ms@redhat.com
+      CLUSTER1_CONFIG: name=rhods-aws-ipi;base_domain=aws.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.13;compute-machine-type=g4dn.2xlarge;timeout=90m;replicas=2;fips=true;ocm-env=production
```

Comment:
please use cluster prefix and not a fix name

Reply:
done